simulation.cpp

```diff
@@ -474,7 +474,7 @@ void test_simulation() noexcept//!OCLINT test may be many
     }
 
 
-#define FIX_ISSUE_39
+//#define FIX_ISSUE_39
```

Comment:
I do not think the test runs forever, but very liklely for a very long time. I will check it out now